images/win/scripts/Tests/SQLPowerShellTools.Tests.ps1

```diff
@@ -0,0 +1,14 @@
+# Validate the installation
+Describe 'Validate SQLPowerShellTools' {
+  BeforeAll {
+    $env:PSModulePath = Get-SystemVariable "PSModulePath"
+  }
+  It 'The SQLPS module exists' {
```

Comment:
there is no need in "exists" word. Just name test case by exact module `It 'SQLPS'`

Reply:
I'm not sure 'It' should be one word. It is 'Assert' part of AAA Unit Testing pattern and expected to describe 'expectation'

Reply:
If test does only "<tool> --version", I suggest to name it like binary name to be consistent with existing tests. Example: https://github.com/actions/virtual-environments/blob/main/images/win/scripts/Tests/Tools.Tests.ps1#L15
If we have more than 1 test-case. Rich naming makes sense